kouncil-frontend/apps/kouncil/src/app/login/auth.service.ts

```diff
@@ -0,0 +1,31 @@
+import {Injectable} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {map} from 'rxjs/operators';
+import {Observable} from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+
+  authenticated: boolean = false;
```

Comment:
should be private BehaviourSubject - public observable to subscribe in components/guards